src/ingest_validation_tests/ome_tiff_validator.py

```diff
@@ -9,11 +9,13 @@ class OmeTiffValidator(Validator):
     cost = 1.0
     def collect_errors(self) -> List[str]:
         rslt = []
-        for glob_expr in ['**/*.ome.tiff', '**/*.OME.TIFF']:
+        for glob_expr in ['**/*.ome.tif', '**/*.ome.tiff', '**/*.OME.TIFF', '**/*.OME.TIF']:
             for path in self.path.glob(glob_expr):
                 try:
                     with tifffile.TiffFile(path) as tf:
-                        metadata = xmltodict.parse(tf.ome_metadata)  # @Notused
+                        xml_document = xmlschema.XmlDocument(tf.ome_metadata)
```

Comment:
@jswelling this looks fine. 

Any reason why you are not using

* bftools
* or python-bioformats

out of curiosity.